### DISCUSSION TITLE
Is it possible to disable the fan "search mode" of the indoor unit?

### DISCUSSION THREAD

**Discussion by wurmtal868** [revk/xESP32-Faikout#268] [Q&A]
I'm using home assistant to control my room temperature by manipulating the set temp of the indoor unit. When the set temperature is lower than the set temperature the indoor unit will turn on the fan every 8-10 minutes to check if there is no heat bubble around the units. Depending on where air stream is located this "search mode" can be uncomfortable. 
Is there an option to disable this "searching" ?
Os is there way to move the airflow in a specific direction where it does not hit people in the room?
![image](https://github.com/revk/ESP32-Faikin/assets/32965910/7e45ba23-7d26-4b33-8efb-e40d2b0b61f1)

**Comment by revk** [comment]
Not one I have heard of, but as always if you find an S21 command for it I can add.

**Reply by MUN0X** [reply]
I’ve always suspected that this behaviour can be changed via the `set_demand_studyinfo` command in the official daikin api. 
Never took the time to figure it out though…
